test/support/fixtures/document_fixtures/test_code.py

```diff
@@ -0,0 +1,25 @@
+class PetShop:
+    """Represents a pet shop with inventory and sales functionality."""
+
+    def __init__(self, name):
+        self.name = name
+        self.inventory = {}
+
+    def add_pet(self, pet_type, quantity):
+        """Adds a specified quantity of a pet type to the inventory."""
```

Comment:
I guess it's kinda nice for our purposes that python doc blocks (or whatever they call them) are _inside_ the methods 🤔

Reply:
yeah, it would be a pain otherwise